src/containers/smart-collection/store/sagas.ts

```diff
@@ -117,10 +117,24 @@ function* fetchRemixables() {
   const tracks: TrackMetadata[] = yield call(
     Explore.getRemixables,
     currentUserId,
-    COLLECTIONS_LIMIT
+    75 // limit
   )
 
-  const processedTracks: Track[] = yield call(processAndCacheTracks, tracks)
+  // Limit the number of times an artist can appear
+  const artistLimit = 3
+  const artistCount: Record<number, number> = {}
+
+  const filteredTracks = tracks.filter(trackMetadata => {
```

Comment:
this looks like it would work, but we should probably take the most popular (first) 3 not just any 3 since the filter will go in any order

probably just a for of loop would be better

Reply:
Does `Array.filter` not go in the order? I always assumed that it did?

Reply:
im wrong